index.d.ts

```diff
@@ -0,0 +1,108 @@
+declare module 'dicom-parser' {
+ 
+  export type ByteArray = Uint8Array | Buffer;
+
+  export interface Element {
+    tag: string;
+    vr?: string;
+    length: number;
+    dataOffset: number;
+    items?: Element[];
+    dataSet?: DataSet;
+    parser?: ByteArrayParser;
+  }
+
+  export interface DataSet {
+    byteArray: ByteArray;
+    byteArrayParser : ByteArrayParser;
+    elements: {
+      [key: string]: Element;
+    };
+    warnings: string[];
+
+    /**
+     * Finds the element for tag and returns an unsigned int 16 if it exists and has data. Use this function for VR type US.
+     */
+    uint16: (tag: string, index?: number) => number;
+
+    /**
+     * Finds the element for tag and returns an signed int 16 if it exists and has data. Use this function for VR type SS.
+     */
+    int16: (tag: string, index?: number) => number;
+
+    /**
+     * Finds the element for tag and returns an unsigned int 32 if it exists and has data. Use this function for VR type UL.
+     */
+    uint32: (tag: string, index?: number) => number;
+
+    /**
+     * Finds the element for tag and returns an signed int 32 if it exists and has data. Use this function for VR type SL.
+     */
+    int32: (tag: string, index?: number) => number;
+
+    /**
+     * Finds the element for tag and returns a 32 bit floating point number if it exists and has data. Use this function for VR type FL.
+     */
+    float: (tag: string, index?: number) => number;
+
+    /**
+     * Finds the element for tag and returns a 64 bit floating point number if it exists and has data. Use this function for VR type FD.
+     */
+    double: (tag: string, index?: number) => number;
+
+    /**
+     * Returns the Value Multiplicity of an elements - the number of values in a multi-valued element.
```

Comment:
`the Value Multiplicity` should read: `the actual Value Multiplicity` to be clear with regards to the DICOM standard. I am not sure what this function does in the case of VR=UN. Also `an elements` => `an element`

Reply:
Changed the description and fixed the typo.

For VR=UN, wouldn't the function still return the actual number of values separated by the backslash delimiter, based on its string representation?